### PR TITLE
#65 Update Gradle

### PR DESCRIPTION
**Pull request by BorysChajdas**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -42,6 +42,10 @@ android {
 			excludes += "/META-INF/{AL2.0,LGPL2.1}"
 		}
 	}
+	compileOptions {
+		sourceCompatibility = JavaVersion.VERSION_17
+		targetCompatibility = JavaVersion.VERSION_17
+	}
```
Comment (MarekMacko) [inline app/build.gradle.kts:48]: Try to apply the same settings for lint module

Comment (BorysChajdas) [reply app/build.gradle.kts:48]: Actually, it is not a problem with a Gradle configuration – locally everything is working fine. The problem is with our Github Actions configuration, the setup java is wrong configured so the build action is using the default Java from the Linux distro (which is 11, that's why it was working before 😄).

Comment (BorysChajdas) [reply app/build.gradle.kts:48]: @MarekMacko Now it works! The problem lay elsewhere than we expected 😄

Comment (MarekMacko) [reply app/build.gradle.kts:48]: Always expect unexpected 😄

**Review by MarekMacko — APPROVED** [review]

**Review by marcinWisnia — APPROVED** [review]

**Pull request merged**